backends/python/server/text_embeddings_server/models/jina_v4.py

```diff
@@ -0,0 +1,81 @@
+import os
+from typing import List
+
+import torch
+from opentelemetry import trace
+from transformers import AutoModel
+
+from text_embeddings_server.models import Model
+from text_embeddings_server.models.types import Embedding, PaddedBatch
+
+tracer = trace.get_tracer(__name__)
+
+
+class JinaV4Model(Model):
```

Comment:
why not implement JinaV4Model in candle? Its much faster than torch here, actually (because you have to use PaddedBatch)